### PR TITLE
update details link url and text

### PR DESCRIPTION
**Pull request by rosiegrant**

### CODE REVIEW

File: directanswercards/documentsearch-standard/component.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
     return {
       value: answer.value,
       snippet: snippet && Formatter.highlightField(snippet.value, snippet.matchedSubstrings), // Text snippet to include alongside the answer
-      viewDetailsText: relatedItemData.title, // Text below the direct answer and snippet
-      viewDetailsLink: relatedItemData.url, // Link for the "view details" text
+      viewDetailsText: relatedItemData.fieldValues.name, // Text below the direct answer and snippet
```
Comment (tmeyer2115) [inline directanswercards/documentsearch-standard/component.js:20]: I think we will need to add a null check here. Could we do something like:
```
const relatedItemData = relatedItem.data || {};
const fieldValues = relatedItemData.fieldValues || {};
```

Comment (rosiegrant) [reply directanswercards/documentsearch-standard/component.js:20]: good call! would this work instead?
`viewDetailsText: relatedItemData.fieldValues && relatedItemData.fieldValues.name, // Text below the direct answer and snippet`

(could get confusing redefining `fieldValues`; we could do `relatedItemDataFieldValues = relatedItemData.fieldValues || {};` but just gets a little long)

Comment (tmeyer2115) [reply directanswercards/documentsearch-standard/component.js:20]: Sure, the `&&` things sounds fine!

Comment (rosiegrant) [reply directanswercards/documentsearch-standard/component.js]: discussed over slack, former is good!

**Review by tmeyer2115 — APPROVED** [review]

**Pull request merged**